google/cloud/forseti/common/util/logger.py

```diff
@@ -78,6 +84,18 @@ def set_logger_level(level):
     LOGLEVEL = level
     _map_logger(lambda logger: logger.setLevel(level))
 
+
+def enable_console_log():
+    """Enable console logging."""
```

Comment:
Can you please clarify this comment to describe this is enabling console logging for existing handlers?

Reply:
Updated.